run.go

```diff
@@ -21,6 +21,9 @@ type Runs interface {
 	// List all the runs of the given workspace.
 	List(ctx context.Context, workspaceID string, options *RunListOptions) (*RunList, error)
 
+	// List all the runs of the given organization.
+	ListForOrganization(ctx context.Context, organisation string, options *RunListForOrganizationOptions) (*RunList, error)
```

Comment:
Minor nitpick: we use the American spelling `organization`

Reply:
Oops! Thank you for spotting this, I fixed it everywhere!